core/src/datayoga_core/utils.py

```diff
@@ -86,8 +88,8 @@ def unescape_field(field: str) -> str:
 
 
 def get_connection_details(connection_name: str, context: Context) -> Dict[str, Any]:
-    if context:
-        connection = context.properties.get("connections").get(connection_name)
+    if context and context.properties:
```

Comment:
what does this change do?

Reply:
how is it related to this PR?

Reply:
saw it and since it's a tiny fix, I decided to fix it.